README.md

```diff
@@ -11,4 +11,4 @@ and use it 3 times inside the `App` instead of static markup.
 ## Instructions
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_person/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://andriy-shymkiv.github.io/react_person/) and add it to the PR description.
```

Comment:
your link is still not work
and I don't see github pages branch

Reply:
@Artem66 do not approve tasks that don't have working demo link